practice-app/backend/disasterresponse/src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 # Details for our datasource
 spring.datasource.url=jdbc:postgresql://localhost:5432/disasterresponse
 spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.username=postgres
```

Comment:
Please change back to the original username and password

Reply:
Done. Thanks.